src/app/course-item/course-item.component.html

```diff
@@ -0,0 +1,14 @@
+<header class="course__header">
+    <h2>{{course.name}}</h2>
+    <section class="course__header-info">
+        <p>{{course.duration}} minutes </p>
+        <p>&nbsp;{{course.date.getDate()}}.{{course.date.getMonth()}}.{{course.date.getFullYear()}}</p>
+    </section>
+</header>
+<section class="course__content">
+    <article>{{course.description}}</article>
+    <aside class="course__buttons">
+        <button class="course__button" (click)="edit()">Edit</button>
+        <button class="course__button" (click)="delete(course.id)">Delete</button>
```

Comment:
The `delete` handler has access to `course.id` from its context. So, it makes no sense to complicate the template passing useless arguments